db/seeds.rb

```diff
@@ -43,7 +43,7 @@
   Category.find_or_create_by(name: "Programming languages")
 
   # create 350 users
-  46.times do |n|
+  400.times do |n|
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.<br>Inconsistent indentation detected.